domains/adoqet.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "AdoQeT",
+        "email": "adoqet@gmail.com"
+    },
+    "record": {
+        "TXT": "log"
```

Comment:
TXT records alone do nothing

Reply:
What am I supposed to write? I don't know anything about it.